definitions/3.0.0/operationReply.json

```diff
@@ -0,0 +1,26 @@
+{
+  "type": "object",
+  "additionalProperties": false,
+  "patternProperties": {
+    "^x-[\\w\\d\\.\\x2d_]+$": {
+      "$ref": "http://asyncapi.com/definitions/3.0.0/specificationExtension.json"
+    }
+  },
+  "properties": {
+    "address": {
+      "oneOf": [
+        {
+          "$ref": "http://asyncapi.com/definitions/3.0.0/Reference.json"
+        },
+        {
+          "$ref": "http://asyncapi.com/definitions/3.0.0/operationReplyAddress.json"
+        }
+      ]
+    },
+    "channel": {
+      "$ref": "http://asyncapi.com/definitions/3.0.0/Reference.json"
+    }
```

Comment:
@magicmatatjahu `messages` field is missing. See https://github.com/asyncapi/spec/blob/next-major-spec/spec/asyncapi.md#operation-reply-object